src/Microsoft.ML.TimeSeries/TimeSeriesProcessing.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.ML.TimeSeriesProcessing
     internal static class TimeSeriesProcessingEntryPoints
     {
         [TlcModule.EntryPoint(Desc = ExponentialAverageTransform.Summary, UserName = ExponentialAverageTransform.UserName, ShortName = ExponentialAverageTransform.ShortName)]
-        public static CommonOutputs.TransformOutput ExponentialAverage(IHostEnvironment env, ExponentialAverageTransform.Arguments input)
+        internal static CommonOutputs.TransformOutput ExponentialAverage(IHostEnvironment env, ExponentialAverageTransform.Arguments input)
```

Comment:
>internal [](start = 8, length = 8)

This appears to already be part of an internal class, so the changes in this particular file were not helpful. Probably revert.

Reply:
Sure but these functions were showing up in API list but I could be wrong as well ... will do in the next iteration.

---
In reply to: [252812773](https://github.com/dotnet/machinelearning/pull/2344#discussion_r252812773) [](ancestors = 252812773)